From 48046924c192ec469d399196869ab49227a3f41b Mon Sep 17 00:00:00 2001
From: Damiano galassi <damiog@gmail.com>
Date: Thu, 1 Dec 2022 15:42:05 +0100
Subject: [PATCH] Revert "pkg-config: Use Libs.private in pkg-config files for
 LIBS_PRIVATE" Breaks GitHub CI build on Ubuntu 20.04.

This reverts commit 27f43f8feab503c86f6022d7ff209b1d749fa13e.
---
 Source/Lib/Decoder/pkg-config.pc.in | 3 +--
 Source/Lib/Encoder/pkg-config.pc.in | 3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/Source/Lib/Decoder/pkg-config.pc.in b/Source/Lib/Decoder/pkg-config.pc.in
index da330911..021dbffb 100644
--- a/Source/Lib/Decoder/pkg-config.pc.in
+++ b/Source/Lib/Decoder/pkg-config.pc.in
@@ -6,7 +6,6 @@ libdir=@SVT_AV1_LIBDIR@
 Name: SvtAv1Dec
 Description: SVT (Scalable Video Technology) for AV1 decoder library
 Version: @DEC_VERSION_MAJOR@.@DEC_VERSION_MINOR@.@DEC_VERSION_PATCH@
-Libs: -L${libdir} -lSvtAv1Dec
-Libs.private: @LIBS_PRIVATE@
+Libs: -L${libdir} -lSvtAv1Dec @LIBS_PRIVATE@
 Cflags: -I${includedir}/svt-av1@DEC_PKG_CONFIG_EXTRA_CFLAGS@
 Cflags.private: -UEB_DLL
diff --git a/Source/Lib/Encoder/pkg-config.pc.in b/Source/Lib/Encoder/pkg-config.pc.in
index ccd71472..47f55686 100644
--- a/Source/Lib/Encoder/pkg-config.pc.in
+++ b/Source/Lib/Encoder/pkg-config.pc.in
@@ -6,7 +6,6 @@ libdir=@SVT_AV1_LIBDIR@
 Name: SvtAv1Enc
 Description: SVT (Scalable Video Technology) for AV1 encoder library
 Version: @ENC_VERSION_MAJOR@.@ENC_VERSION_MINOR@.@ENC_VERSION_PATCH@
-Libs: -L${libdir} -lSvtAv1Enc
-Libs.private: @LIBS_PRIVATE@
+Libs: -L${libdir} -lSvtAv1Enc @LIBS_PRIVATE@
 Cflags: -I${includedir}/svt-av1@ENC_PKG_CONFIG_EXTRA_CFLAGS@
 Cflags.private: -UEB_DLL
-- 
2.37.1 (Apple Git-137.1)

